src/OpenTelemetry.Exporter.Geneva/Metrics/Transport/MetricEtwDataTransport.cs

```diff
@@ -24,6 +27,9 @@ private MetricEtwDataTransport()
     }
 
     [NonEvent]
+#if NET6_0_OR_GREATER
```

Comment:
@eerhardt - will need suppress on `SendOtlpProtobufEvent` below on line 45 as well.

Reply:
This is a good catch. It looks like we need to update the Aot test app to catch these. Will update.

Reply:
Fixed.